ocs_ci/deployment/vmware.py

```
@@ -1351,7 +1349,7 @@ def deploy(self, log_cli_level="DEBUG"):
                 # multus deployment need more time to generate CSRs
                 is_multus_enabled = config.ENV_DATA.get("is_multus_enabled")
                 csr_timeout = (
-                    4800 if is_multus_enabled else (2400 if num_nodes >= 6 else 1500)
+                    4800 if is_multus_enabled else (2400 if num_nodes >= 6 else 2400)
```

Ruff (0.15.20)

[warning] Useless `if`-`else` condition

(RUF034)